README.md

```diff
@@ -22,13 +16,11 @@ The components and reference information for interoperability used in this proje
 
 #### CHT
 
-The structure of documents in the CHT database reflect the configuration of the system, and therefore do not map directly to a FHIR message format. To achieve interoperability we used a middleware to convert the CHT data structure into a standardized form so the other systems can read it.
+The structure of documents in the CHT database reflect the configuration of the system, and therefore do not map directly to a FHIR message format. To achieve interoperability we used a middleware to convert the CHT data structure into a standardized form so the other systems can read it. Below is the standard data workflow;
 
 ![](./docs/images/flow.png)
 
-This project uses OpenHIM as the middleware component with [Mediators](http://openhim.org/docs/configuration/mediators/) to do the conversion. [Outbound Push](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/outbound/) is configured to make a request to the middleware when relevant documents are created or modified in the CHT. A Mediator then creates a FHIR resource which is then routed to OpenHIM. OpenHIM routes the resource to any other configured systems.
```

Comment:
Super weird, my comments from yesterday disappeared; I'm not sure why. I saw some changes in the GitHub interface this morning that might be connected. 🤔 

I think you should keep this paragraph as it explains nicely (and in a technical way) the architecture image above. I don't find it redundant nor overly complicated.

Reply:
Done